​
# Sourcegraph v3.33 upgrade issue <!-- Ticket Title  Hint: include keywords to make it searchable -->

- Zendesk Link: [#6998](https://sourcegraph.zendesk.com/agent/tickets/6998)
- Application engineer: Alex Jean-Baptiste
- Customer: Goldman Sachs <!-- Redact if this contains personally identifying information -->
- Date: Mar 18

<!-- Data populated from integration, speak to Ben Gordon or Michael Bali if not working -->
<!-- During Internal team trial, fill missing data manually (we are waiting for all data to sync) -->

## Technical Environment
- Version: ​3.33.0
- Deployment: Kubernetes
- External Services: gitlab, cvs
- Auth Providers: saml


## Links
<!-- Data for application engineer manual entry -->
- Slack Links: N/A
- GitHub Issue Link: N/A
- Doc Update Link: N/A

## Summary
### Description of Customer Issue
The customer received the following error:
​Post "http://gitserver-0.gitserver:3178/repos": dial tcp: lookup gitserver-0.gitserver on 10.100.0.10:53: no such host in the UI. 
​
### Troubleshooting Steps
​

Searched through recently resolved tickets for this customer and saw that the issue had a previous resolution involving provisioning more resources for gitserver
Communicated to the customer that they should check their Grafana dashboard for signs intensive resource load on the gitserver service


### Resolution
​
The customer lowered the gitserver replica count from 2 to 1 which reduced resource usage and resolved the issue.

### Relevant Docs Pages Used/Created
None.
​
### Relevant Error / Logs
<!-- Please redact keys, tokens, and personal identifying information -->
​"log": "2022-03-17T18:31:42.500132735Z stderr F {\"level\":\"warn\",\"ts\":1647541902.5000348,\"caller\":\"channelz/logging.go:75\",\"msg\":\"[core]grpc: addrConn.createTransport failed to connect to {jaeger-collector:14250 jaeger-collector:14250 <nil> 0 <nil>}. Err: connection error: desc = \\\"transport (file:///%22transport): Error while dialing dial tcp: lookup jaeger-collector on 10.100.0.10:53: read udp 172.31.154.53:36889->10.100.0.10:53: i/o timeout\\\". Reconnecting...\",\"system\":\"grpc\",\"grpc_log\":true}",
  "kubernetes": {
      "pod_name": "gitserver-0",
      "namespace_name": "sourcegraph",
      "pod_id": "94a8eca4-68be-44ca-a1c9-accbe0dd509b",
      "labels": {
          "app": "gitserver",
          "controller-revision-hash": "gitserver-84b6d4899b",
          "deploy": "sourcegraph",
          "group": "backend",
          "statefulset.kubernetes.io/pod-name": "gitserver-0",
          "type": "gitserver"
      },
      "annotations": {
          "kubernetes.io/psp": "eks.privileged",
          "vpc.amazonaws.com/pod-eni": "[{\"eniId\":\"eni-09291fc59f2df90d5\",\"ifAddress\":\"16:00:d5:65:7d:2d\",\"privateIp\":\"172.31.133.16\",\"vlanId\":7,\"subnetCidr\":\"172.31.128.0/19\"}]"
      },
      "host": "ip-172-31-146-141.ec2.internal",
      "container_name": "jaeger-agent",
      "docker_id": "fca36f7598fdcf17c557ac8f771d1caa4abb70fa4c7b20c7237d5b2305a79007",
      "container_hash": "816887984493.dkr.ecr.us-east-1.amazonaws.com/sourcegraph/jaeger-agent@sha256:9f12c673eea43fae3b2da9fef9c0cad0a909a1f78c436013d3b65d69a298ebc3",
      "container_image": "sha256:485a871a868d84bbb9f11ae5be93bc0095606b406202e3c454bbe7b75795bad9"
  }
}

​
​
<!-- Once complete, upload a copy to https://github.com/sourcegraph/support-tools-internal/tree/main/resolved-tickets as a .md file -->
<!-- Name the file 6998.md -->
